app/locales/en.json

```diff
@@ -21,12 +21,18 @@
     "button_text": "Import past locations",
     "google": {
       "disclaimer": "Safe Paths has no affiliation with Google and never shares your data.",
-      "instructions_detailed": "Visit Google Takeout and export your Location History using the following settings: \n1. Delivery method: \"Add to Drive\" \n2. Frequency: \"Export once\" \n3. File type & size: \".zip\" and \"1GB\"\n4. Google sends an email when the export is ready \n5. Return here to import locations. Import options:\n- Import from Google Drive\n- Download from browser, then import from local phone files. Make sure to be on WiFi network as files can be big.",
+      "instructions_detailed": "Visit Google Takeout and export your Location History using the following settings: \n1. Delivery method: \"Add to Drive\" \n2. Frequency: \"Export once\" \n3. File type & size: \".zip\" and \"1GB\"\n4. Google sends an email when the export is ready \n5. Return here to import locations from Google Drive",
       "instructions_first": "Adding location data from Google will give you a head start on building your recent locations.",
       "instructions_second": "Before you can import, you must first \"Take out\" your location data from Google.",
       "title": "Google Maps",
-      "visit_button_text": "Visit Google Takeout"
+      "visit_button_text": "Visit Google Takeout",
+      "already_imported":"Provided Takeout file has already been imported.",
```

Comment:
Please add screenshots of these errors/status messages, so I can upload to the i18n platform, thanks!